src/App.jsx

```diff
@@ -2,14 +2,22 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount((currentCount) => {
+      const result = currentCount + 1;
+
+      return result;
+    });
   };
 
   const add100 = () => {
```

Comment:
To previous student comment, please fix
```suggestion
  const add100 = () => setCount(currentCount => currentCount + 100)
```